.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,25 +5,25 @@ Please add testing instructions to projects/plugins/jetpack/to-test.md in a new
 
 Fixes #
 
-#### Changes proposed in this Pull Request:
+## Changes proposed in this Pull Request:
```

Comment:
@jeherve What do you think about changing this to:

```
## Proposed changes:
```

Reply:
I like it, short and to the point!